library/src/main/java/com/yayandroid/locationmanager/providers/locationprovider/GooglePlayServicesLocationProvider.java

```diff
@@ -106,94 +114,97 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     @Override
-    public void onConnected(Bundle bundle) {
-        LogUtils.logI("GoogleApiClient is connected.");
-        boolean locationIsAlreadyAvailable = false;
+    public void onConnected() {
+        LogUtils.logI("Start request location updates.");
 
-        if (getConfiguration().googlePlayServicesConfiguration().ignoreLastKnowLocation()) {
```

Comment:
I don't see how this is related to any of the other changes? Please keep the `lastKnowLocation` logic preserved here.